### PR TITLE
simple token for mobile

### LINKED ISSUE #611: "Select another booking" doesn't work in mobile layout with simple token
When I open MMB with a simple token on mobile, then click on "Select another booking", it's not working properly.

Please, implement one component for this button with all necessary logic to not duplicate the code in `bookingItem/Header.js` and `MobileBookingDetail.js`

### PR DESCRIPTION
**Pull request by jalbertsr**
Close #611 <br/><br/><br/><url>LiveURL: https://smartfaq-611-simple-token-for-mobile.surge.sh</url>

### CODE REVIEW

File: src/MobileBookingHeader/MobileUserDetail.js [hunk 1]
```diff
@@ -4,8 +4,9 @@ import * as React from 'react';
 import css from 'styled-jsx/css';
 import { Text } from '@kiwicom/orbit-components';
 
-import { UserContext, type UserContextType } from '../context/User';
+import { UserContext } from '../context/User';
 import SignOutButton from '../common/layout/SignOutButton';
+import type { UserContextType } from '../context/User';
 
 const MobileUserDetailStyle = css`
```
Comment (jalbertsr) [inline src/MobileBookingHeader/MobileUserDetail.js]: For the user detail section of the mobile header when the user uses simple token we don't have the name of the user account, we only have the names of the passengers in the booking.

What should we show instead? 

full login:
screenshot

simple token login: 
screenshot

@jaroslav-kubicek

Comment (kantord) [reply src/MobileBookingHeader/MobileUserDetail.js]: How about hiding the entire tab altogether? Does the logout button have a purpose?

Comment (kantord) [reply src/MobileBookingHeader/MobileUserDetail.js]: From the two options I prefer the second one.

Comment (jalbertsr) [reply src/MobileBookingHeader/MobileUserDetail.js]: good point! you shouldn't be available to sign out, because you're not fully logged in

Comment (kantord) [reply src/MobileBookingHeader/MobileUserDetail.js]: Well, just make sure also that it's not possible to compromise user data. What if I log in on a publicly used device? Will an unauthorized person be able to download my boarding passes/tickets and see my personal data on them?

Comment (kantord) [reply src/MobileBookingHeader/MobileUserDetail.js]: Maybe it's FE's responsibility though

**Pull request merged**